server/routes/offenceCodeDecisions/offenceData.ts

```diff
@@ -0,0 +1,8 @@
+// Data retained while progressing through the questions to decide and offenceCode. At the end of the decisions when
+// we have an offence code the data here should be enough to uniquely determine the questions and answers given.
+export type OffenceData = {
+  victimStaff?: string
+  victimPrisoner?: string
+  victimOtherPerson?: string
+  offenceCode?: string
```

Comment:
Can this be made in to a number? is it a number?

Reply:
Coming from url, so saves us parsing. We have to convert eventually, so not such a problem.